### PR TITLE
Enhancement: Use Travis for running a few things here and there

### PR DESCRIPTION
**Pull request by localheinz**
This PR adds a very simple `.travis.yml` configuration which
    
* [x] self-updates Composer in `before_install` section
* [x] installs dependencies with Composer in `install` section
* [x] runs `phpspec` in `script` section
* [x] runs the build against PHP 5.4, 5.5, and 5.6

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: php
+
+php:
+  - 5.4
```
Comment (kayladnls) [inline .travis.yml]: We need to up that to at least 5.6, no?

Comment (localheinz) [inline .travis.yml]: @kayladnls 

Not sure, are we using any PHP 5.6 features yet?

Comment (kayladnls) [inline .travis.yml]: Can we add both 5.4, 5.5 and 5.6?

Comment (localheinz) [inline .travis.yml]: @kayladnls 

:+1:

**Comment by localheinz** [comment]
Nice!

**Pull request merged**